src/components/BezierCurveEditor/BezierCurveEditorEndPoints.tsx

```diff
@@ -20,29 +29,80 @@ export function BezierCurveEditorEndPoints({
   strokeWidth,
   handleLineColor,
   fixedHandleColor,
+  ...props
 }: Props) {
   const { startCoordinate, endCoordinate } = bezierCurveParamsFromSizeAndValue(size, value);
 
+  if (props.isEditable !== true) {
+    return (
+      <>
+        <span
+          className={cx(styles.handle, styles.fixed)}
+          style={{
+            top: `${startCoordinate[1] + outerAreaSize + strokeWidth}px`,
+            left: `${startCoordinate[0] + strokeWidth}px`,
+            borderColor: handleLineColor,
+            backgroundColor: fixedHandleColor,
+          }}
+        ></span>
+        <span
+          className={cx(styles.handle, styles.fixed)}
+          style={{
+            top: `${endCoordinate[1] + outerAreaSize + strokeWidth}px`,
+            left: `${endCoordinate[0] + strokeWidth}px`,
+            borderColor: handleLineColor,
+            backgroundColor: fixedHandleColor,
+          }}
+        ></span>
+      </>
+    );
+  }
+
+  const {
+    movingEndPoint,
+    movingStartPoint,
+    handleEndPointPointerLeaveOrUp,
+    handleEndPointPointerMove,
+    handleEndPointStartMoving,
+    handleStartPointPointerLeaveOrUp,
+    handleStartPointPointerMove,
+    handleStartPointStartMoving,
+  } = props;
+
   return (
     <>
-      <span
-        className={cx(styles.handle, styles.fixed)}
+      <button
+        type="button"
+        className={cx(styles.handle, styles.fixed, {
+          [styles.active]: movingStartPoint,
+          // [startHandleActiveClassName]: !!startHandleActiveClassName && movingStartPoint,
```

Comment:
I suppose we could add new active classname props for the fixed node handles?

Reply:
Updated.